specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,111 @@
+= Specification for adding disk details in node hardware inventory
+
+Add disk details also as a part of node hardware inventory
+
+== Problem description
+
+Currently disk details are missing in node hardware inventory. Add disk details
+in node hardware inventory and disk details should contain device_name, fs_type,
+fs_uuid, model, mount_point, parent, size, name, disk_type, used, ssd, vendor,
+disk_id, node_id.
```

Comment:
There is an issue of disk names getting remapped when one of the disks are removed from existing inventory. @brainfunked was talking about some disk-id/fsid maintained by kernel can be used to maintain the uniqueness of the disk. This needs to figured out and added to this spec